build-caching-maven-samples/kotlin-project/pom.xml

```diff
@@ -106,6 +106,10 @@
                                         <fileSets>
                                             <fileSet>
                                                 <name>sourceDirs</name>
+                                                <normalization>
+                                                    <strategy>RELATIVE_PATH</strategy>
```

Comment:
i'm surprised we needed this, was it absolute path before :O 

Can you also add/test `ignoreEmptyDirectories` if it isn't already present?

Reply:
Yep you're correct, it's not needed - `RELATIVE_PATH` is the default strategy. Updated to remove the `RELATIVE_PATH` specification.

Reply:
Thanks for removing the `RELATIVE_PATH`. What was the outcome on `ignoreEmptyDirectories`?

Reply:
As expected, adding an empty directory to the source tree caused a cache miss. I've updated the PR to apply empty directory normalization as well.

Reply:
Thanks for checking this!